php-practice/section1.php

```diff
@@ -193,5 +247,15 @@
 // Does the element at index 'id' get overwritten? What does that tell you?
```

Comment:
What is your response to these questions? :-) you can just leave the response in a  git hub comment